### ISSUE TITLE
Importing Stratego files that import `libstratego-aterm` causes build failures

### ISSUE THREAD

**Issue by molenzwiebel** [metaborg/spoofax-pie#100]
## Summary
Using `ppdebug` from `libspoofax/stratego/debug` results in PIE build errors because it cannot find `libstratego-aterm`.

## What you did
```
module foo

imports
  strategolib
  libspoofax/stratego/debug

rules
  foo = ![1, 2, 3]; ppdebug(|"Test")
```

## What you expected to happen
Pretty printed ATerm in the console.

## What actually happened
Build error due to unresolved `libstratego-aterm`. Directly importing libstratego-aterm fails too.

## Context
* Spoofax version: 0.16.11

**Comment by Gohla** [comment]
This is probably due to `libstratego-aterm` not being included as a built-in library, and it is not yet possible to add additional built-in libraries from `spoofaxc.cfg` yet. It can be added by default by changing https://github.com/metaborg/spoofax-pie/blob/develop/lwb/metalang/cfg/cfg/src/main/java/mb/cfg/metalang/CfgStrategoSource.java#L46, or added through `spoofaxc.cfg` by creating syntax for it that adds to that list.

**Comment by molenzwiebel** [comment]
Fixed in 0.16.15.
